src/game/turret.ts

```diff
@@ -70,6 +70,20 @@ export function constructNetTurret(
   EM.ensureComponentOn(e, SyncDef);
   e.sync.dynamicComponents.push(YawPitchDef.id);
 
+  // setup camera params
+  EM.ensureComponentOn(e, CameraFollowDef, 0);
+  setCameraFollowPosition(e, "thirdPersonOverShoulder");
+  quat.rotateY(
+    e.cameraFollow.rotationOffset,
+    e.cameraFollow.rotationOffset,
+    Math.PI / 2
+  );
+  quat.rotateX(
+    e.cameraFollow.rotationOffset,
+    e.cameraFollow.rotationOffset,
+    -Math.PI / 8
+  );
```

Comment:
Notice how now we just set up these offsets once at turret creation, and then no where else in turret do we futz with cameras (except to set the priority on man/unman)